spec/lib/myinfo/v4/security_spec.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+require 'jwe'
+
+describe MyInfo::V4::Security do
+  describe '#create_code_verifier' do
+    it 'creates a code verifier of the correct length' do
+      code_verifier = described_class.create_code_verifier
+      expect(code_verifier).not_to be_empty
+    end
+  end
+
+  describe '#create_code_challenge' do
+    # code_verifier test case values taken from MyInfo's code challenge generator
+    let(:code_verifier) { 'U2ptdDRDMkNhYmF6MVlESTZuWXZ2dkVyMXhiVDNhZko' }
+    let(:expected_code_challenge) { 'FkMwoc-HA4015MuSfEjjjPvxF0e82VxGHYP4Q8-12b4' }
+    it 'returns the correct code_challenge' do
```

Comment:
nit: should have a blank line after the last let. try running rubocop before commits.